skimage/feature/util.py

```diff
@@ -76,9 +77,9 @@ def plot_matched_features(
         The Axes object where the images and their matched features are drawn.
     keypoints_color : matplotlib color, optional
         Color for keypoint locations.
-    matches_color : matplotlib color, optional
-        Color for lines which connect keypoint matches. By default the
-        color is chosen randomly.
+    matches_color : matplotlib color or sequence of matplotlib color, optional
```

Comment:
What do you think about scratching the option to pass a single color? Users's can still easily use a single color if they want, and it simplifies the API?

Reply:
I also thought about this. I think it comes down to convenience vs simplified API and simplified code. It is hard for me tell whether there are many people using this function with a single fixed color.

I have no hard opinion on this. You or anybody else can decide what is the best option to implement.

Reply:
See https://github.com/scikit-image/scikit-image/pull/7541#discussion_r1760546529. Not worth deprecating IMO.